fgaddressestocsv.py

```diff
@@ -72,37 +77,53 @@ def parse(fd):
 	with open(fd,'rb') as fd_input:
 		for line in fd_input:
 			line = line.lstrip().rstrip().strip()

```

Comment:
For this to work for me in python3, I needed to change this to include ascii decode, like: 

```
     line = line.decode('ascii').lstrip().rstrip().strip()
```